File: 6-kyu/multiples-of-3-or-5/javascript/solution.js
function solution(number){
  let sumAllNumbers = 0;
  let counter = 0;
  if (number < 0)
    return 0
  while (counter < number)
  {
    if (counter % 3 == 0 || counter % 5 == 0)
      sumAllNumbers += counter;
    counter++;
  }
  return sumAllNumbers
}

File: 6-kyu/multiples-of-3-or-5/ruby/solution.rb
def solution(number)
  sumAllNumbers = 0
  counter = 0
  (number < 0) ? (return 0) : nil
  while (counter < number)
    ((counter % 3 == 0) || (counter % 5 == 0)) ? sumAllNumbers+=counter : nil
    counter+=1
  end
  return sumAllNumbers
end